modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/metadata/DefaultMetadataContext.java

```diff
@@ -107,4 +139,14 @@ public ClassTypeLoader getTypeLoader() {
   public BaseTypeBuilder getTypeBuilder() {
     return BaseTypeBuilder.create(JAVA);
   }
+
+  @Override
+  public Optional<ExpressionLanguageMetadataService> getExpressionLanguageMetadataService() {
```

Comment:
`@since`

Reply:
The `@since` is in the `InternalMetadataContext` interface.